playbooks/README.md

```diff
@@ -46,7 +46,7 @@ practice should be incorporated, submit a PR!
 | [How to run a Knowledge Share](/playbooks/running-knowledge-share.md#readme) | How to handle the people process for Knowledge Shares |
 | [System Criticality](/playbooks/system-criticality.md#readme) | A framework for treating systems differently according to how critical they are |
 | [Taming notifications](/playbooks/taming-notifications.md#readme) | How to set up notifications so that they leave you alone outside of working hours |
-| [Technical planning](/playbooks/technical-planning.md#readme) | How to write technical plans at Artsy |
+| [Technology choices](/playbooks/technical-planning.md#readme) | How to write technical plans at Artsy |
```

Comment:
Error? (The tech choices doc is already listed below.)

Reply:
Strange! Not sure how the linting would have touched that, but I'll push up a fix.

Reply:
Ah, that must have been inferred from this frontmatter when the linter ran. I'll update that…

https://github.com/artsy/README/blob/ca1c2357b073863512e7deff57f669713404788f/playbooks/technical-planning.md?plain=1#L1-L4